pyke/meta.yaml

```diff
@@ -11,7 +11,8 @@ source:
     sha1: 6f6bfab243564c17218d81fa40bc3bf24e0d5847  # [py3k]
 
 build:
-    number: 1
+    number: 1  # [not osx]
+    number: 2  # [osx]
```

Comment:
i'm chasing a bug in the osx build for python3
it looked like the pyke build artifact for py3 osx was a python2 version.  I put this in to see if this is the case

looking at the travis build, the test failure has moved, so there may be something here

I'm happy to revert or change this prior to merging, ponce we've worked out what's going wrong